### DISCUSSION TITLE
Dudas Práctica 1

### DISCUSSION THREAD

**Discussion by NiiSV811** [Launch-X-Latam/MisionFrontEnd#101] [Q&A]
Hasta cuando tenemos plazo para entregar la Práctica 1?
Por donde se debe entregar?
Se califica o algo parecido?

**Comment by JosafatJimenezB** [comment]
Por lo mientras realiza la practica y guárdala en un repositorio de GitHub por que al final del reto frontend te piden un link con los repos de las practicas para poder pasar a la mision backend

**Reply by MaferMtz** [reply]
Sabes si ponen un examen de certificacion?

**Reply by MaferMtz** [reply]
O solo te califican con las practicas?
